VogtPayroll2/EmployeeHourly1.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace VogtPayroll2
+{
+    class EmployeeHourly : Employee
+    {
+        public EmployeeHourly(string aName, int aHoursWorked, decimal aPayRate)
+            : base(aName, aHoursWorked, aPayRate)
+        {
+            this.name = aName;
```

Comment:
Because you call `base()`, you do not need to set the parameters that are being set by base.

But note in my other comment I said to remove `payRate` from the base class (since you have it being calculated via an abstract method)